Dockerfiles/admin/Dockerfile

```diff
@@ -134,7 +129,6 @@ RUN { \
  && chown -R "${OPENCAST_USER}:${OPENCAST_GROUP}" "${OPENCAST_HOME}" "${OPENCAST_DATA}" \
   \
   # Cleanup
- && apk del --no-cache nodejs \
```

Comment:
Is this being used at runtime? otherwise, we should still remove nodejs after compilation.